src/agent/agent.c

```diff
@@ -516,10 +516,12 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
         }
 
         value = cfg_get_value(agent->config, CFG_MANAGER_HOST);
-        if (value) {
-                if (!agent_set_host(agent, value)) {
-                        return false;
-                }
+        if (!value) {
+                value="127.0.0.1";
```

Comment:
I think the value of the default host should defined next to the default port in [protocol.h](https://github.com/containers/hirte/blob/main/src/libhirte/common/protocol.h#L6).